src/OversetSimulation.cpp

```diff
@@ -127,9 +127,11 @@ void OversetSimulation::run_timesteps(const int add_pic_its, const int nsteps)
         std::to_string(tstart));
 
     for (int nt = tstart; nt < tend; ++nt) {
+        using ClockT = std::chrono::steady_clock;
```

Comment:
We could use the `tick` `tock` functionality for this and the stuff below? Or is there a reason not to?

Reply:
Yeah I will look at it again.